### PR TITLE
Add testsuite for createListOfSize

### LINKED ISSUE #29: Add test suite for createListOfSize util
**Please describe the motive behind this issue.**
Need a new test suite for createListOfSize util method.

**Describe the solution you'd like to propose**
Create a new test suite containing relevant test cases to test the createListOfSize util.

**Describe alternatives you've considered**
NA

**Additional context**
Update the util in case any edge case is not covered by the current implementation.

### PR DESCRIPTION
**Pull request by selamanse**
# Pull Request Template

## Description

Adds Testsuite for createListOfSize

Fixes #29 

## Type of change

Please mark the relevant options.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

running `npm run test:ci`

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: src/utils/createListOfSize.test.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+test('createListOfSize 4 and begin with 2', () => {
+    expect(createListOfSize(4, 2)).toEqual([2, 3, 4, 5]);
+});
+
+test('createListOfSize 4 and begin with 0', () => {
+    expect(createListOfSize(4, 0)).toEqual([0, 1, 2, 3]);
+});
+
+test('createListOfSize 4 and begin with -4', () => {
```
Comment (DemonDaddy22) [inline src/utils/createListOfSize.test.ts:11]: @selamanse test case description doesn’t match the test case, -4 must be -2.

File: src/utils/createListOfSize.test.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    expect(createListOfSize(4, -2)).toEqual([-2, -1, 0, 1]);
+});
+
+test('createListOfSize 999 and begin with -999', () => {
+    const testList = createListOfSize(999, -998);
+    expect(testList.length).toEqual(999);
+    expect(testList[998]).toEqual(0);
+});
```
Comment (DemonDaddy22) [inline src/utils/createListOfSize.test.ts:19]: Can you also add a test case when the second argument is omitted, it should default to 1 in such a case?

**Review by DemonDaddy22 — CHANGES_REQUESTED** [review]

**Review by DemonDaddy22 — APPROVED** [review]
LGTM

**Comment by selamanse** [comment]
@DemonDaddy22 thx what about the restyled bot? I cannot fix it via `npm run lint:fix`

**Comment by DemonDaddy22** [comment]
@selamanse It will automatically close once this PR gets merged. It just fixes the code quality issues.

**Comment by DemonDaddy22** [comment]
@all-contributors please add @selamanse for code.
